stdlib/logging/__init__.pyi

```diff
@@ -31,13 +31,27 @@ class Filterer(object):
     def removeFilter(self, filter: _FilterType) -> None: ...
     def filter(self, record: LogRecord) -> bool: ...
 
+class Manager(object):
+    root: RootLogger
+    disable: int
+    emittedNoHandlerWarning: bool
+    loggerDict: dict[str, Type[Logger]]
```

Comment:
Values can't be classes (`Type[Logger]`), but they can be `Logger` instances or `PlaceHolder` instances:

```suggestion
    loggerDict: dict[str, Union[Logger, PlaceHolder]]
```